libnavigation-core/src/test/java/com/mapbox/navigation/core/MapboxNavigationTest.kt

```diff
@@ -195,48 +190,18 @@ class MapboxNavigationTest {
         verify(exactly = 1) { tripSession.unregisterAllVoiceInstructionsObservers() }
     }
 
-    @Test
-    fun fasterRoute_timerStarted() {
```

Comment:
Faster route and route refresh are not required for navigation. Move this to be tested independently